resources/build/build-triggers.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+#
+# This file maps specific paths to build triggers
+#
+
+# the base folder for each pattern does not need to be included, nor oem folders
+# e.g. android='common/predictive-text|common/lexical-model-types'
+# will expand into android='^(android|(oem/[^/]+/android)|common/predictive-text|common/lexical-model-types)'
+
+watch_android='android|common/predictive-text|common/lexical-model-types'
+watch_ios='common/predictive-text|common/lexical-model-types'
+watch_linux='common/engine'
+watch_mac='common/engine'
```

Comment:
Is this for future feature when mac uses Keyman core?

Reply:
Yes, also to force builds whenever Keyman core is built (this reminds me that I want to rename that folder to common/core!)